Gemfile

```diff
@@ -52,9 +52,9 @@ group :development, :test do
 end
 
 group :test do
-  gem 'rspec-rails', require: false
-  gem 'factory_girl', require: false
-  gem 'capybara', require: false
+  gem 'rspec-rails'
+  gem 'factory_girl_rails'
+  gem 'capybara'
```

Comment:
I may not be the most efficient reviewer, but perhaps that will be a good thing. "require:false" seems crufty, but can you briefly point me to a reason it's been changed?